wasmedge-sys/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "wasmedge-rs"
```

Comment:
@nhynes 
Maybe it is good to use `wasmedge-sys`?
The crate `wasmedge-rs` is occupied by trademark squatter, not by us.

Reply:
it's supposed to be that. good catch.

> occupied by trademark squatter, not by us.
yeah. 👎  to squatsmer